perllib/FixMyStreet/App/Controller/Waste/Bulky.pm

```diff
@@ -214,11 +215,17 @@ sub index_booking : Private {
 sub amend : Chained('setup') : Args(1) {
     my ($self, $c, $id) = @_;
 
+    my $small = $c->stash->{small_items};
     $c->stash->{first_page} = 'intro';
-    $c->stash->{form_class} = 'FixMyStreet::App::Form::Waste::Bulky::Amend';
+    $c->stash->{form_class} ||= $small
```

Comment:
You've already set it to SmallItems::Amend below if it's small items.
This can just be `$c->stash->{form_class} ||= 'FixMyStreet::App::Form::Waste::Bulky::Amend';`